generator.js

```diff
@@ -11,23 +11,24 @@ const moodTwo = 'happy'
 const moodThree = 'sleepy'
 const moodFour = 'sad'
```

Comment:
At this point, we are declaring constant strings just to embed them in an array of strings. We'd just be better off hard coding them there.